modules/i18n/src/models/namespaces/console-ns.ts

```diff
@@ -4010,32 +4116,209 @@ export interface ConsoleNS {
                     };
                 };
                 notifications: {
-                    addUser: Notification;
-                    addUserPendingApproval: Notification;
+                    addUser: {
+                        error?: {
+                        message: string;
+                        description: string;
+                        genericMessage?: string;
+                     }
+                        genericError?: {
+                        message: string;
+                        description: string;
+                        genericMessage?: string;
+                     }
+                        success?: {
+                        message: string;
+                        description: string;
+                        genericMessage?: string;
+                     }
+                     }
+                     ;
+                    addUserPendingApproval: {
+                        error?: {
+                        message: string;
+                        description: string;
+                        genericMessage?: string;
+                     }
+                        genericError?: {
+                        message: string;
+                        description: string;
+                        genericMessage?: string;
+                     }
+                        success?: {
+                        message: string;
+                        description: string;
+                        genericMessage?: string;
+                     }
+                     }
+                     ;
                     bulkImportUser: {
                         validation: {
-                            emptyRowError: NotificationItem;
```

Comment:
Any particular reason to get rid of common interfaces and repeat it accross the namespace file?

Reply:
To prevent redundantly making attributes in common interfaces optional due to different changes in features using these common interfaces. Attributes can be defined separately for each feature by this approach.